MITRE ATT&CK for Azure Sentinel.ipynb

```diff
@@ -0,0 +1,2063 @@
+{
```

Comment:
<p>Can't get figs to show in Lab but work fine in normal Notebook env.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>requires jupyterlab-plotly extension. added comments.</p>